lib/app_files/category_i_files/borrowed_books.dart

```diff
@@ -12,72 +15,88 @@ class BorrowedBooks extends StatefulWidget {
 class _BorrowedBooks extends State<BorrowedBooks> {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        backgroundColor: AppColors.BASE_COLOR,
-        centerTitle: true,
-        title: TextHeader(text: 'Borrowed Books'),
-      ),
-      body: SingleChildScrollView(
-        child: Column(
-          children: [
-            ListView.builder(
-                physics: NeverScrollableScrollPhysics(),
-                shrinkWrap: true,
-                itemCount: 10,
-                itemBuilder: ((context, index) {
-                  return listContainer();
-                }))
-          ],
-        ),
+    return SingleChildScrollView(
```

Comment:
i believe this method will work well instead of the deleted one. and it work more precisely.